jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -833,6 +833,10 @@ public DateField(long seconds, HttpField dateField)
 
     private static class DynamicErrorHandler extends ErrorHandler {}
 
+    /**
+     * The {@link Context} associated with all {@link Request}s prior to being handled by a
+     * {@link ContextHandler}.
```

Comment:
Expand here, saying what is particular about `ServerContext`, in particular regarding the context path (and perhaps something else?).